app/adminPage.html

```diff
@@ -87,7 +80,7 @@ <h4 class="search-heading">Filter Results</h4>
                             <input id="endDate" type="date">
                         </div>
                     </div>
-                    <table class="table user-score-table">
+                    <table class="table tableStyles">
```

Comment:
`tableStyles`? really - you know better than that, this is not even remotely semantic!